cms/static/js/i18n/ar/djangojs.js

```diff
@@ -871,7 +870,7 @@
     "If you do not yet have an account, use the button below to register.": "\u0625\u0630\u0627 \u0644\u0645 \u062a\u0643\u0646 \u062a\u0645\u0644\u0643 \u062d\u0633\u0627\u0628\u064b\u0627 \u0628\u0639\u062f\u060c \u064a\u064f\u0631\u062c\u0649 \u0627\u0633\u062a\u062e\u062f\u0627\u0645 \u0627\u0644\u0632\u0631\u0651 \u0623\u062f\u0646\u0627\u0647 \u0644\u0644\u062a\u0633\u062c\u064a\u0644.", 
     "If you don't verify your identity now, you can still explore your course from your dashboard. You will receive periodic reminders from %(platformName)s to verify your identity.": "\u0625\u0630\u0627 \u0644\u0645 \u062a\u0624\u0643\u0651\u062f \u0647\u0648\u064a\u0651\u062a\u0643 \u0627\u0644\u0622\u0646\u060c \u0633\u064a\u0628\u0642\u0649 \u0628\u0625\u0645\u0643\u0627\u0646\u0643 \u0627\u0633\u062a\u0643\u0634\u0627\u0641 \u0645\u0633\u0627\u0642\u0643 \u0645\u0646 \u062e\u0644\u0627\u0644 \u0644\u0648\u062d\u0629 \u0645\u0639\u0644\u0648\u0645\u0627\u062a\u0643. \u0648\u0633\u062a\u062a\u0644\u0642\u0651\u0649 \u0631\u0633\u0627\u0626\u0644 \u062a\u0630\u0643\u064a\u0631\u064a\u0629 \u062f\u0648\u0631\u064a\u0651\u064b\u0627 \u0645\u0646 %(platformName)s \u0644\u062a\u0623\u0643\u064a\u062f \u0647\u0648\u064a\u0651\u062a\u0643. ", 
-    "If you don't verify your identity now, you can still explore your course from your dashboard. You will receive periodic reminders from {platformName} to verify your identity.": "\u0625\u0630\u0627 \u0644\u0646 \u062a\u0642\u0645 \u0628\u0627\u0644\u062a\u062d\u0642\u0642 \u0645\u0646 \u0647\u0648\u064a\u062a\u0643 \u0627\u0644\u0622\u0646\u060c \u0644\u0627 \u064a\u0632\u0627\u0644 \u0628\u0625\u0645\u0643\u0627\u0646\u0643 \u0627\u0633\u062a\u0643\u0634\u0627\u0641 \u0627\u0644\u062f\u0648\u0631\u0629 \u0645\u0646 \u062e\u0644\u0627\u0644 \u0644\u0648\u062d\u0629 \u0627\u0644\u0645\u0639\u0644\u0648\u0645\u0627\u062a \u0627\u0644\u062e\u0627\u0635\u0629 \u0628\u0643. \u0633\u0648\u0641 \u062a\u062a\u0644\u0642\u0649 \u0631\u0633\u0627\u0626\u0644 \u062a\u0630\u0643\u064a\u0631 \u062f\u0648\u0631\u064a\u0629 \u0645\u0646 {platformName} \u0644\u0644\u062a\u062d\u0642\u0642 \u0645\u0646 \u0647\u0648\u064a\u062a\u0643.", 
```

Comment:
This string hasn't changed. Why we need to update its translation?

Reply:
Translation reverted